src/test/unit/test_spans_container.py

```diff
@@ -204,6 +204,51 @@ def test_add_tag():
     ]
 
 
+def test_start_manual_trace_simple_flow():
+    SpansContainer.get_span().start_manual_trace("11")
+    SpansContainer.get_span().stop_manual_trace("11")
+
+    manual_tracers = SpansContainer.get_span().function_span[MANUAL_TRACES_KEY]
+    now = datetime.now().timestamp() * 1000
+    one_min_ago = (datetime.now() - timedelta(minutes=1)).timestamp() * 1000
```

Comment:
the interesting thing to check here is not "one minute ago", but just capture the timestamp in the beginning of the test. (just put `before= get_current_ms_time()` as the first line of the test)